public/data/food/ingredients.json

```diff
@@ -16,6 +15,8 @@
     "inedible_part": 0.2,
     "land_occupation": 4.46758,
     "name": "Oeuf Bleu-Blanc-Cœur FR Conv.",
+    "processId": "375de455-cd49-51f7-8ecf-58a3e626bc73",
+    "processSourceId": "AGRIBALU000000003105149",
```

Comment:
Ad we can get the source id through the linked process, is it a good idea t duplicate the information in two different files?

Reply:
Debatable, I thought it could be useful. But I've got no strong opinion on this and since we're gonna have a backoffice that can do this soon, let's just put what's necessary